### PR TITLE
Feature/filter on flexiscatter

### PR DESCRIPTION
**Pull request by wendyyang**
add tissue and combination filter

### CODE REVIEW

File: index.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 from app import app
 from pages import project, home, matrix, combinations
 from page_components import header
-from utils import url_parser
+from utils import url_parser, tissues
```
Comment (Donnyvdm) [inline index.py]: 'tissues' is not used, please remove

File: utils.py [hunk 2]
```diff
@@ -291,3 +291,8 @@ def get_combination_matrices_summary(project_id, lib1_id, lib2_id, percentiles):
         .filter(models.MatrixResult.lib2_id == lib2_id)
     return pd.read_sql(query.statement, session.get_bind())\
         .describe(percentiles=percentiles)
+
+def tissues():
```
Comment (Donnyvdm) [inline utils.py]: the name 'tissues' is not descriptive enough for this function, please change to something like 'get_all_tissues()'

File: index.py (hunk 1 above)
Comment (Donnyvdm) [inline index.py]: 'tissues' is not used, please remove

File: components/project_scatter.py [hunk 3]
```diff
@@ -107,7 +152,8 @@ def update_scatter(x_axis_field, y_axis_field, color_field, project_id):
     fig_data = summary
     return {
         'data': [
-            go.Scattergl(
+            #go.Scattergl(
```
Comment (Donnyvdm) [inline components/project_scatter.py]: Please remove commented-out line

File: components/project_scatter.py [hunk 4]
```diff
@@ -1,78 +1,112 @@
...
 @app.callback(
     dash.dependencies.Output('project-scatter', 'figure'),
     [dash.dependencies.Input('x-axis-select', 'value'),
      dash.dependencies.Input('y-axis-select', 'value'),
      dash.dependencies.Input('color-select', 'value'),
+     dash.dependencies.Input('tissue-select', 'value'),
+     dash.dependencies.Input('combination-select', 'value'),
      dash.dependencies.Input('project-id', 'children')])
-def update_scatter(x_axis_field, y_axis_field, color_field, project_id):
+def update_scatter(x_axis_field, y_axis_field, color_field, tissues, combinations, project_id):
```
Comment (Donnyvdm) [inline components/project_scatter.py]: The variable names should be more descriptive. Perhaps something like 'selected_tissues' and 'selected_combinations'

Comment (howard-lightfoot) [reply components/project_scatter.py]: I
```suggestion
def update_scatter(x_axis_field, y_axis_field, color_field, selected_tissues, selected_combinations, project_id):
```

File: components/project_scatter.py (hunk 3 above)
Comment (howard-lightfoot) [reply components/project_scatter.py]:
```suggestion
```

File: index.py (hunk 1 above)
Comment (howard-lightfoot) [reply index.py]:
```suggestion
from utils import url_parser
```

File: utils.py (hunk 2 above)
Comment (howard-lightfoot) [reply utils.py]:
```suggestion
def get_all_tissues():
```

**Pull request merged**